src/sentry/conf/server.py

```diff
@@ -2864,3 +2864,11 @@ def build_cdc_postgres_init_db_volume(settings):
 
 # How long we should wait for a gateway proxy request to return before giving up
 GATEWAY_PROXY_TIMEOUT = None
+
+SENTRY_SLICING_LOGICAL_PARTITION_COUNT = 256
+# This maps a dataset for slicing by name and (lower logical partition, upper physical partition)
+# to a given slice. A slice is a set of physical resources in Sentry and Snuba.
+#
+# For each dataset, the range [0, SENTRY_SLICING_LOGICAL_PARTITION_COUNT) must be mapped
+# to a slice ID
+SENTRY_SLICING_CONFIG: Mapping[str, Mapping[Tuple[int, int], int]] = {}
```

Comment:
I don't know if dataset is the right terminology for Sentry. It feels more like a storage term. What if we state these mappings in terms of entities or something else?